ChangeLog.md

```diff
@@ -20,6 +20,10 @@ See docs/process.md for more on how version tagging works.
 
 3.1.48 (in development)
 -----------------------
+- The minimum version of node required to run the compiler was updated from
+  10.19.0 to 16.20.0.  One side effect of this is that the
+  `NODEJS_CATCH_REJECTION` settings no longer relevant and is will be ignored.
+  (#20551)
```

Comment:
Please update the PR title to mention that this is for the compiler, and not for the generated JS.